index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Pacifico&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Varela+Round&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="icon" href="/images/favicon.ico" />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>Baseball Project</title>
+  </head>
+
+  <body>
+    <section>
+      <div class="image-box">
+        <img src="images/vanilla_coding_logo.png" />
+      </div>
+      <h1>Baseball</h1>
+
+      <!-- 야구게임 Start -->
+      <script>
+    
+      const initAnswer = () => {
+        const flags = Array(10).fill(false);
+      
+        let answer = '';
+        while (answer.length < 3) {
+          const random = parseInt(Math.random() * 10);
+      
+          if (!flags[random]) {
+            answer += random;
+            flags[random] = true;
+          }
+        }
+        return answer;
+      }
```

Comment:
```suggestion
      };
```
화살표 익명 함수를 변수에 담아서 사용하셨는데, 이 경우 함수 표현식에 해당하게 됩니다.
함수 표현식은 구문 마지막에 세미콜론을 붙여주어야 합니다.
아래 링크를 꼭 참고해서 조사해보세요.

**함수 표현식 vs 함수 선언문** 
- https://ko.javascript.info/function-expressions